apps/desktop/src/autofill/services/desktop-autofill.service.ts

```diff
@@ -81,6 +81,20 @@ export class DesktopAutofillService implements OnDestroy {
     this.listenIpc();
   }
 
+  async AdHocSync(): Promise<any> {
```

Comment:
Should this be camel case?
```suggestion
  async adHocSync(): Promise<any> {
```